### DISCUSSION TITLE
Feature-Wish: support of non-gregorian calendars

### DISCUSSION THREAD

**Discussion by porthd** [nhn/tui.calendar#1362] [Q&A] [answered]
There are many non-gregorian calendar-systems in the world. There exists in example the islamic calendar, the chinese calendar or the hebrew calendar. It would be nice, to make available that calendars in the presentation to help for the localization of websites.

Is there a chance, to get this feature in the future?

The recalculation of the calendars should not be the great problem. See (german text) http://www.nabkal.de/umrechnung.html

P.S. Sorry about my poor english.

**Comment by adhrinae** [comment, marked as answer]
Unfortunately, we don't have any plan to support for non-gregorian calendars.

**Reply by porthd** [reply]
Thanks for your quick answer. Sorry for my late response. I think, your decision is right. It is really complex, to make the views flexible for different calendar-systems.

Then I have to calculate a mapping-table between the gregorian-calandar and the non-gregorian-calendar and publish the mapping as a eventlist, if I want to show the date of one day in a notation of an other non-gregorian calendar. Or is there a better way to inform the reader of a date about this?
